src/collectors.m4

```diff
@@ -409,6 +409,7 @@ m4_define([_DISCARD_VALUES_FOR_ALL_ARGUMENTS], [m4_do(
 	[m4_define([_POSITIONALS_INF], 0)],
 	[m4_define([HAVE_POSITIONAL_VARNUM], 0)],
 	[m4_define([HAVE_DOUBLEDASH], 0)],
+	[m4_define([HAVE_SUPPLIED], [])],
```

Comment:
This statement is ineffective, as `HAVE_SUPPLIED` is used as an identifier of a list, and it is not a macro - the list itself lives as a macro named differently. There is no `m4_list_create` because it is not needed, so please remove this line.